trendmicro_deepsecurity/icon_trendmicro_deepsecurity/util/shared.py

```diff
@@ -0,0 +1,34 @@
+
+from komand.exceptions import PluginException
+
+def tryJSON(response):
```

Comment:
Can you utilize type hinting and annotations?

Reply:
done